.github/CODEOWNERS

```diff
@@ -478,6 +478,7 @@ modules/test/playwright/tests/export-import-web    @liferay-headless
 modules/test/playwright/tests/headless-builder-impl    @liferay-headless
 modules/test/playwright/tests/headless-builder-web    @liferay-headless
 modules/test/playwright/tests/layout-set-prototype-web    @liferay-headless
+modules/test/playwright/tests/portal-impl/portlet    @liferay-frontend
```

Comment:
I'm a bit nervous about this, I wouldn't want us to take responsibility for everything in `portal-impl` :sweat_smile: 

Does SF require this? If so, I suppose it's fine, we can tweak it later on.